### PR TITLE
adding an * by required fields in forms

### PR DESCRIPTION
**Pull request by lyndaidaii**
Adding an * by required fields for Key Name

Before changes:
UI-d

After changes: 
Ui_LI
Addresses https://github.com/NuGet/NuGetGallery/issues/8181

### CODE REVIEW

File: src/Bootstrap/less/theme/page-api-keys.less [hunk 1]
```diff
@@ -53,6 +53,11 @@
         height: 282px;
       }
     }
+
+    .required:after{
```
Comment (joelverhagen) [inline src/Bootstrap/less/theme/page-api-keys.less:57]: image

**Comment by joelverhagen** [comment]
Would it be possible to have a little more space before the asterisk?

Also, how does the screen reader behave for this case? /cc @ryuyu Is it correct?

**Comment by ryuyu** [comment]
As far as I can tell, it looks like what is expected. Let's make sure that what screen reader reads makes sense, and I think we are good.

**Review by joelverhagen — APPROVED** [review]
Address comments then :shipit:

**Comment by loic-sharma** [comment]
+1 for adding space, it looks a little compact to me currently.

**Review by loic-sharma — APPROVED** [review]

**Comment by lyndaidaii** [comment]
screen reader reads 'the key name is required' when we click the input box of key name.

**Pull request merged**